File: 01 - Orientação a objetos/Conceitos/Orientacao_Objetos/src/exemplo01/Pessoa.java
package exemplo01;

import javax.swing.JOptionPane;

public class Pessoa {
	
	// Atributos
	String nome;
	int idade;
	double altura;
	
	// Método para poder obter os dados
	void obterDados() {
		nome = JOptionPane.showInputDialog("Informe seu nome");
		idade = Integer.parseInt(JOptionPane.showInputDialog("Digite sua idade"));
		altura = Double.parseDouble(JOptionPane.showInputDialog("Informe sua altura:"));
		
	}
	
	// Método para retornar os dados
	void retornarDados() {
		String mensagem = "Nome: " + nome;
			   mensagem += "\nIdade: " + idade;
			   mensagem += "\nAltura: " + altura;
			   
		JOptionPane.showMessageDialog(null, mensagem);
	}

}

File: 05 - JSP/Conceitos/src/main/webapp/pessoa.js
function valida(){
	// Obter nome e idade
	let nome = document.getElementById("nome").value;
	let idade = parseInt(document.getElementById("idade").value);
	
	// Condicional
	if(nome == ""){
		alert("Informe o nome!");
		return false;
	} else if(idade <0 || idade > 120) {
		alert("idade inválida!");
		return false;
	} else {
		return true;
	}
	
	
}